h/presenters/annotation_searchindex.py

```diff
@@ -6,7 +6,7 @@
 class AnnotationSearchIndexPresenter(AnnotationBasePresenter):
     """Present an annotation in the JSON format used in the search index."""
 
-    def __init__(self, annotation, request):
+    def __init__(self, annotation, request):  # pylint: disable=super-init-not-called
```

Comment:
`AnnotationBasePresenter` takes an `annotation_resource`

Reply:
This whole thing is an incredible mess which hopefully I'll be looking at soon.